src/rapidcsv.h

```diff
@@ -1068,6 +1068,12 @@ namespace rapidcsv
         throw std::out_of_range("column name row index < 0: " + std::to_string(mLabelParams.mColumnNameIdx));
       }
 
+      // increase table size if necessary:
+      int rowIdx = mLabelParams.mColumnNameIdx;
+      if ((int)mData.size() <= rowIdx) mData.resize(rowIdx + 1);
```

Comment:
Current code is not wrong, but can we change the order (so it matches other rapidcsv code), i.e.
```if (rowIdx > (int)mData.size())```

Reply:
Also can we change the code formatting to something like:
```
if (rowIdx > (int)mData.size())
{
  mData.resize(rowIdx + 1);
}
```